OWNERS

```diff
@@ -1,18 +0,0 @@
-approvers:
-  - ecordell
-  - njhale
-  - dinhxuanvu
-  - kevinrizza
-  - benluddy
-  - gallettilance
-  - anik120
-  - exdx
-  - awgreene
-  - hasbro17
-  - ankitathomas
-  - joelanford
-  - timflannagan
-reviewers:
-  - timflannagan
-  - ankitathomas
-component: "OLM"
```

Comment:
It looks like this got picked up by accident. We'll need to revert this and remove the nested OWNERS file in the staging/api directory.

Reply:
Actually, let's revert the change to the root OWNERS file, but tackle updating the root .gitingore in a separate PR based on [Ben's comment](https://github.com/openshift/operator-framework-olm/pull/37#issuecomment-814330580).

Reply:
Good catch there, thanks Tim!